dashboard/app/bisect_test.go

```diff
@@ -188,11 +188,13 @@ For information about bisection process see: https://goo.gl/tpsmEJ#bisection
 
 		c.expectEQ(msg.To, []string{
 			"author@kernel.org",
-			"bugs@syzkaller.com",
-			"default@maintainers.com",
 			"reviewer1@kernel.org",
 			"reviewer2@kernel.org",
 		})
+		c.expectEQ(msg.Cc, []string{
+			"bugs@syzkaller.com",
+			"default@maintainers.com",
```

Comment:
Using the new users, `default@maintainers.com` should go to `To` as we check for other lists on `pkg/`. The old user stores LKML on `cfg.DefaultMaintainers` which goes to Cc. So `default@maintainers.com` would be on To if I change the test to receive `Commit.Recipients`

Reply:
`cfg.DefaultMaintainers` is part of the configuration file for the dashboard app, which the user needs to specify to run the app.  The production config for https://syzkaller.appspot.com has `DefaultMaintainers: []string{"linux-kernel@vger.kernel.org"}`.  This config will be the same both before and after your change.

I guess this isn't too important since the list from `get_maintainer.pl` will include `linux-kernel` too.  But I think there's an edge case where a guilty file or commit isn't identifiable, so `syzkaller-bugs` and `linux-kernel` will be the only recipients.  In that case they'll both get put on `Cc` initially.  That will trigger the fallback where both get placed on `To`.

That's okay, though I'd argue that out of principle the `To` should always be kernel lists or people, not `syzkaller-bugs`, as the responsibility for fixing the bugs lies (or should lie) with kernel developers.

So if you can make it always do the expected thing, please do so.  But if it would add much extra complexity, I think what you currently have is okay.  Can you please make sure it's properly tested, though?